<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Swift Mailing List Archive</title>
    <link rel="stylesheet" href="/css/app-13f065ae5e595562a5022c544e3b976c.css?vsn=d">
  </head>

  <body>
    <div class="container">
      <header class="header">
        <img src="/images/swift-d0237fc716ba0932a940049990beba1b.svg?vsn=d" height="70">
      </header>

      <p class="alert alert-info" role="alert"></p>
      <p class="alert alert-danger" role="alert"></p>

    </div> <!-- /container -->
    <main role="main">
<div class="comment-wrapper"><ul class="comments"><li class="comment"><div class="avatar"><img src="https://www.gravatar.com/avatar/aea2e26d03f169b1ad6008a77bab533f?s=50"></div><header><strong>ERROR: test_breakpoint_callback_dwarf</strong> from <string>SonWa</string> &lt;six at sonwa.cn&gt;<p>August 29, 2016 at 05:00:00pm</p></header><div class="content"><p>Hi,<br>system:CentOS7.2<br></p><p><br>run test error (./swift/utils/build-script --preset=buildbot_linux install_destdir=/tmp/install installable_package=/tmp/swift.tar.gz)<br> ===================================================<br> <br>--- Finished tests for swift ---<br></p><p>+ mkdir -p /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/test-results<br></p><p>+ pushd /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/test-results<br></p><p>/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/test-results /usr/local/src<br></p><p>+ env SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc SWIFTLIBS=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/lib/swift /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang --results-formatter lldbsuite.test_event.formatter.xunit.XunitFormatter --results-file /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/test-results/results.xml -O--xpass=success -O--xfail=success -O-ndsym -O-rdebugserver &#39;-O-rlibc\\+\\+&#39; -O-rlong.running -O-rbenchmarks &#39;-O-rrequires.one?.of.darwin&#39;<br></p><p>Testing: 658 test suites, 2 threads<br></p><p> 27 out of 658 test suites processed - TestExprsChar.py                       <br></p><p><br></p><p><br>Session logs for test failures/errors/unexpected successes will go into directory &#39;2016-08-28-05_59_45&#39;<br></p><p>Command invoked: /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 42983 --inferior -p TestMultithreaded.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p><br></p><p><br>Configuration: arch=x86_64 compiler=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang<br></p><p>----------------------------------------------------------------------<br></p><p>Collected 12 tests<br></p><p><br></p><p><br>======================================================================<br></p><p>ERROR: test_breakpoint_callback_dwarf (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the that SBBreakpoint callback is invoked when a breakpoint is hit.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_breakpoint_callback&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp test_breakpoint_callback.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>test_breakpoint_callback.cpp:32: error: undefined reference to &#39;lldb::SBDebugger::SetAsync(bool)&#39;<br></p><p>test_breakpoint_callback.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:38: error: undefined reference to &#39;lldb::SBBreakpoint::SetCallback(bool (*)(void*, lldb::SBProcess&amp;, lldb::SBThread&amp;, lldb::SBBreakpointLocation&amp;), void*)&#39;<br></p><p>test_breakpoint_callback.cpp:41: error: undefined reference to &#39;lldb::SBTarget::LaunchSimple(char const**, char const**, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_breakpoint_callback] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_breakpoint_callback_dwo (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the that SBBreakpoint callback is invoked when a breakpoint is hit.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_breakpoint_callback&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp test_breakpoint_callback.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>test_breakpoint_callback.cpp:32: error: undefined reference to &#39;lldb::SBDebugger::SetAsync(bool)&#39;<br></p><p>test_breakpoint_callback.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:38: error: undefined reference to &#39;lldb::SBBreakpoint::SetCallback(bool (*)(void*, lldb::SBProcess&amp;, lldb::SBThread&amp;, lldb::SBBreakpointLocation&amp;), void*)&#39;<br></p><p>test_breakpoint_callback.cpp:41: error: undefined reference to &#39;lldb::SBTarget::LaunchSimple(char const**, char const**, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_breakpoint_callback] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_breakpoint_callback_gmodules (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the that SBBreakpoint callback is invoked when a breakpoint is hit.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_breakpoint_callback&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp test_breakpoint_callback.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>test_breakpoint_callback.cpp:32: error: undefined reference to &#39;lldb::SBDebugger::SetAsync(bool)&#39;<br></p><p>test_breakpoint_callback.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:38: error: undefined reference to &#39;lldb::SBBreakpoint::SetCallback(bool (*)(void*, lldb::SBProcess&amp;, lldb::SBThread&amp;, lldb::SBBreakpointLocation&amp;), void*)&#39;<br></p><p>test_breakpoint_callback.cpp:41: error: undefined reference to &#39;lldb::SBTarget::LaunchSimple(char const**, char const**, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_breakpoint_callback] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_description_dwarf (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the description of an SBListener breakpoint event is valid.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_description&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_description.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_description.cpp:27: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:28: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_description.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_description.cpp:34: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_description.cpp:35: error: undefined reference to &#39;lldb::SBEvent::GetDescription(lldb::SBStream&amp;)&#39;<br></p><p>test_listener_event_description.cpp:36: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_description] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_description_dwo (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the description of an SBListener breakpoint event is valid.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_description&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_description.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_description.cpp:27: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:28: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_description.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_description.cpp:34: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_description.cpp:35: error: undefined reference to &#39;lldb::SBEvent::GetDescription(lldb::SBStream&amp;)&#39;<br></p><p>test_listener_event_description.cpp:36: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_description] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_description_gmodules (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the description of an SBListener breakpoint event is valid.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_description&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_description.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_description.cpp:27: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:28: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_description.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_description.cpp:34: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_description.cpp:35: error: undefined reference to &#39;lldb::SBEvent::GetDescription(lldb::SBStream&amp;)&#39;<br></p><p>test_listener_event_description.cpp:36: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_description] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_process_state_dwarf (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test that a registered SBListener receives events when a process<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_process_state&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_process_state.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_process_state.cpp:28: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:29: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_process_state.cpp:35: error: undefined reference to &#39;lldb::SBProcess::GetProcessFromEvent(lldb::SBEvent const&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:36: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:38: error: undefined reference to &#39;lldb::SBProcess::GetNumThreads()&#39;<br></p><p>test_listener_event_process_state.cpp:40: error: undefined reference to &#39;lldb::SBStream::Clear()&#39;<br></p><p>test_listener_event_process_state.cpp:41: error: undefined reference to &#39;lldb::SBProcess::GetThreadAtIndex(unsigned long)&#39;<br></p><p>test_listener_event_process_state.cpp:42: error: undefined reference to &#39;lldb::SBThread::GetDescription(lldb::SBStream&amp;) const&#39;<br></p><p>test_listener_event_process_state.cpp:43: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_process_state.cpp:46: error: undefined reference to &#39;lldb::SBThread::GetNumFrames()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBThread::GetFrameAtIndex(unsigned int)&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::GetSymbol() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::GetName() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_process_state] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_process_state_dwo (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test that a registered SBListener receives events when a process<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_process_state&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_process_state.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_process_state.cpp:28: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:29: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_process_state.cpp:35: error: undefined reference to &#39;lldb::SBProcess::GetProcessFromEvent(lldb::SBEvent const&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:36: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:38: error: undefined reference to &#39;lldb::SBProcess::GetNumThreads()&#39;<br></p><p>test_listener_event_process_state.cpp:40: error: undefined reference to &#39;lldb::SBStream::Clear()&#39;<br></p><p>test_listener_event_process_state.cpp:41: error: undefined reference to &#39;lldb::SBProcess::GetThreadAtIndex(unsigned long)&#39;<br></p><p>test_listener_event_process_state.cpp:42: error: undefined reference to &#39;lldb::SBThread::GetDescription(lldb::SBStream&amp;) const&#39;<br></p><p>test_listener_event_process_state.cpp:43: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_process_state.cpp:46: error: undefined reference to &#39;lldb::SBThread::GetNumFrames()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBThread::GetFrameAtIndex(unsigned int)&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::GetSymbol() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::GetName() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_process_state] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_process_state_gmodules (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test that a registered SBListener receives events when a process<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_process_state&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_process_state.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_process_state.cpp:28: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:29: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_process_state.cpp:35: error: undefined reference to &#39;lldb::SBProcess::GetProcessFromEvent(lldb::SBEvent const&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:36: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:38: error: undefined reference to &#39;lldb::SBProcess::GetNumThreads()&#39;<br></p><p>test_listener_event_process_state.cpp:40: error: undefined reference to &#39;lldb::SBStream::Clear()&#39;<br></p><p>test_listener_event_process_state.cpp:41: error: undefined reference to &#39;lldb::SBProcess::GetThreadAtIndex(unsigned long)&#39;<br></p><p>test_listener_event_process_state.cpp:42: error: undefined reference to &#39;lldb::SBThread::GetDescription(lldb::SBStream&amp;) const&#39;<br></p><p>test_listener_event_process_state.cpp:43: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_process_state.cpp:46: error: undefined reference to &#39;lldb::SBThread::GetNumFrames()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBThread::GetFrameAtIndex(unsigned int)&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::GetSymbol() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::GetName() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_process_state] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>----------------------------------------------------------------------<br></p><p>Ran 12 tests in 42.607s<br></p><p><br></p><p><br>RESULT: FAILED (0 passes, 0 failures, 9 errors, 0 skipped, 3 expected failures, 0 unexpected successes)<br></p><p>Session logs for test failures/errors/unexpected successes can be found in directory &#39;2016-08-28-05_59_45&#39;<br></p><p><br></p><p><br>[TestMultithreaded.py FAILED]<br></p><p>Command invoked: /usr/bin/python /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 42983 --inferior -p TestMultithreaded.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p> 29 out of 658 test suites processed - TestFixIts.py                          <br></p><p><br></p><p><br>Session logs for test failures/errors/unexpected successes will go into directory &#39;2016-08-28-05_59_45&#39;<br></p><p>Command invoked: /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 42983 --inferior -p TestFormatters.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p><br></p><p><br>Configuration: arch=x86_64 compiler=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang<br></p><p>----------------------------------------------------------------------<br></p><p>Collected 3 tests<br></p><p><br></p><p><br>python: /usr/local/src/llvm/tools/clang/lib/AST/Decl.cpp:2562: bool clang::FunctionDecl::isReservedGlobalPlacementOperator() const: Assertion `getDeclName().getNameKind() == DeclarationName::CXXOperatorName&#39; failed.<br></p><p><br></p><p><br>[TestFormatters.py FAILED]<br></p><p>Command invoked: /usr/bin/python /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 42983 --inferior -p TestFormatters.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p>185 out of 658 test suites processed - TestPlatformCommand.py                     <br></p><p><br></p><p><br>Session logs for test failures/errors/unexpected successes will go into directory &#39;2016-08-28-05_59_45&#39;<br></p><p>Command invoked: /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 42983 --inferior -p TestPluginCommands.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p><br></p><p><br>Configuration: arch=x86_64 compiler=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang<br></p><p>----------------------------------------------------------------------<br></p><p>Collected 1 test<br></p><p><br></p><p><br>======================================================================<br></p><p>ERROR: test_load_plugin (TestPluginCommands.PluginCommandTestCase)<br></p><p>   Test that plugins that load commands work correctly.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc DYLIB_CXX_SOURCES=&#39;plugin.cpp&#39; LD_EXTRAS=&#39;-shared -L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; DYLIB_NAME=&#39;plugin&#39; CFLAGS_EXTRAS=&#39;-std=c++11 -I/usr/local/src/lldb/include -fPIC&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [libplugin.so] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/functionalities/plugins/commands<br></p><p>----------------------------------------------------------------------<br></p><p>Ran 1 test in 0.612s<br></p><p><br></p><p><br>RESULT: FAILED (0 passes, 0 failures, 1 errors, 0 skipped, 0 expected failures, 0 unexpected successes)<br></p><p>Session logs for test failures/errors/unexpected successes can be found in directory &#39;2016-08-28-05_59_45&#39;<br></p><p><br></p><p><br>[TestPluginCommands.py FAILED]<br></p><p>Command invoked: /usr/bin/python /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 42983 --inferior -p TestPluginCommands.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p>599 out of 658 test suites processed - TestREPLDynamicType.py                                    <br></p><p><br></p><p><br>Session logs for test failures/errors/unexpected successes will go into directory &#39;2016-08-28-05_59_45&#39;<br></p><p>Command invoked: /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 42983 --inferior -p TestREPLThrowReturn.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=1:int<br></p><p><br></p><p><br>Configuration: arch=x86_64 compiler=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang<br></p><p>----------------------------------------------------------------------<br></p><p>Collected 1 test<br></p><p><br></p><p><br></p><p><br></p><p>[TestREPLThrowReturn.py FAILED]<br></p><p>Command invoked: /usr/bin/python /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 42983 --inferior -p TestREPLThrowReturn.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=1:int<br></p><p>658 out of 658 test suites processed - TestRecursiveTypes.py                                     <br></p><p>4 test files marked for rerun<br></p><p><br></p><p><br></p><p><br></p><p>Rerunning the following files:<br></p><p>  api/multithreaded/TestMultithreaded.py<br></p><p>  repl/error_return/TestREPLThrowReturn.py<br></p><p>  functionalities/plugins/commands/TestPluginCommands.py<br></p><p>  expression_command/formatters/TestFormatters.py<br></p><p>rerun will use the &#39;threading&#39; test runner strategy<br></p><p>Testing: 4 test suites, 1 thread<br></p><p>0 out of 4 test suites processed - <br></p><p><br></p><p><br>Session logs for test failures/errors/unexpected successes will go into directory &#39;2016-08-28-05_59_45&#39;<br></p><p>Command invoked: /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 45531 --inferior -p TestMultithreaded.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p><br></p><p><br>Configuration: arch=x86_64 compiler=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang<br></p><p>----------------------------------------------------------------------<br></p><p>Collected 12 tests<br></p><p><br></p><p><br>======================================================================<br></p><p>ERROR: test_breakpoint_callback_dwarf (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the that SBBreakpoint callback is invoked when a breakpoint is hit.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_breakpoint_callback&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp test_breakpoint_callback.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>test_breakpoint_callback.cpp:32: error: undefined reference to &#39;lldb::SBDebugger::SetAsync(bool)&#39;<br></p><p>test_breakpoint_callback.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:38: error: undefined reference to &#39;lldb::SBBreakpoint::SetCallback(bool (*)(void*, lldb::SBProcess&amp;, lldb::SBThread&amp;, lldb::SBBreakpointLocation&amp;), void*)&#39;<br></p><p>test_breakpoint_callback.cpp:41: error: undefined reference to &#39;lldb::SBTarget::LaunchSimple(char const**, char const**, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_breakpoint_callback] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_breakpoint_callback_dwo (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the that SBBreakpoint callback is invoked when a breakpoint is hit.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_breakpoint_callback&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp test_breakpoint_callback.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>test_breakpoint_callback.cpp:32: error: undefined reference to &#39;lldb::SBDebugger::SetAsync(bool)&#39;<br></p><p>test_breakpoint_callback.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:38: error: undefined reference to &#39;lldb::SBBreakpoint::SetCallback(bool (*)(void*, lldb::SBProcess&amp;, lldb::SBThread&amp;, lldb::SBBreakpointLocation&amp;), void*)&#39;<br></p><p>test_breakpoint_callback.cpp:41: error: undefined reference to &#39;lldb::SBTarget::LaunchSimple(char const**, char const**, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_breakpoint_callback] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_breakpoint_callback_gmodules (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the that SBBreakpoint callback is invoked when a breakpoint is hit.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_breakpoint_callback&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp test_breakpoint_callback.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>test_breakpoint_callback.cpp:32: error: undefined reference to &#39;lldb::SBDebugger::SetAsync(bool)&#39;<br></p><p>test_breakpoint_callback.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>test_breakpoint_callback.cpp:38: error: undefined reference to &#39;lldb::SBBreakpoint::SetCallback(bool (*)(void*, lldb::SBProcess&amp;, lldb::SBThread&amp;, lldb::SBBreakpointLocation&amp;), void*)&#39;<br></p><p>test_breakpoint_callback.cpp:41: error: undefined reference to &#39;lldb::SBTarget::LaunchSimple(char const**, char const**, char const*)&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>test_breakpoint_callback.cpp:49: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_breakpoint_callback] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_description_dwarf (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the description of an SBListener breakpoint event is valid.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_description&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_description.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_description.cpp:27: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:28: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_description.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_description.cpp:34: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_description.cpp:35: error: undefined reference to &#39;lldb::SBEvent::GetDescription(lldb::SBStream&amp;)&#39;<br></p><p>test_listener_event_description.cpp:36: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_description] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_description_dwo (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the description of an SBListener breakpoint event is valid.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_description&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_description.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_description.cpp:27: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:28: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_description.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_description.cpp:34: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_description.cpp:35: error: undefined reference to &#39;lldb::SBEvent::GetDescription(lldb::SBStream&amp;)&#39;<br></p><p>test_listener_event_description.cpp:36: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_description] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_description_gmodules (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test the description of an SBListener breakpoint event is valid.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_description&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_description.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_description.cpp:27: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:28: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_description.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_description.cpp:34: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_description.cpp:35: error: undefined reference to &#39;lldb::SBEvent::GetDescription(lldb::SBStream&amp;)&#39;<br></p><p>test_listener_event_description.cpp:36: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:38: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_description.cpp:39: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_description] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_process_state_dwarf (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test that a registered SBListener receives events when a process<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_process_state&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_process_state.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_process_state.cpp:28: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:29: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_process_state.cpp:35: error: undefined reference to &#39;lldb::SBProcess::GetProcessFromEvent(lldb::SBEvent const&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:36: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:38: error: undefined reference to &#39;lldb::SBProcess::GetNumThreads()&#39;<br></p><p>test_listener_event_process_state.cpp:40: error: undefined reference to &#39;lldb::SBStream::Clear()&#39;<br></p><p>test_listener_event_process_state.cpp:41: error: undefined reference to &#39;lldb::SBProcess::GetThreadAtIndex(unsigned long)&#39;<br></p><p>test_listener_event_process_state.cpp:42: error: undefined reference to &#39;lldb::SBThread::GetDescription(lldb::SBStream&amp;) const&#39;<br></p><p>test_listener_event_process_state.cpp:43: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_process_state.cpp:46: error: undefined reference to &#39;lldb::SBThread::GetNumFrames()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBThread::GetFrameAtIndex(unsigned int)&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::GetSymbol() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::GetName() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_process_state] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_process_state_dwo (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test that a registered SBListener receives events when a process<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_process_state&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_process_state.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_process_state.cpp:28: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:29: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_process_state.cpp:35: error: undefined reference to &#39;lldb::SBProcess::GetProcessFromEvent(lldb::SBEvent const&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:36: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:38: error: undefined reference to &#39;lldb::SBProcess::GetNumThreads()&#39;<br></p><p>test_listener_event_process_state.cpp:40: error: undefined reference to &#39;lldb::SBStream::Clear()&#39;<br></p><p>test_listener_event_process_state.cpp:41: error: undefined reference to &#39;lldb::SBProcess::GetThreadAtIndex(unsigned long)&#39;<br></p><p>test_listener_event_process_state.cpp:42: error: undefined reference to &#39;lldb::SBThread::GetDescription(lldb::SBStream&amp;) const&#39;<br></p><p>test_listener_event_process_state.cpp:43: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_process_state.cpp:46: error: undefined reference to &#39;lldb::SBThread::GetNumFrames()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBThread::GetFrameAtIndex(unsigned int)&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::GetSymbol() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::GetName() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_process_state] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>======================================================================<br></p><p>ERROR: test_sb_api_listener_event_process_state_gmodules (TestMultithreaded.SBBreakpointCallbackCase)<br></p><p>    Test that a registered SBListener receives events when a process<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc LD_EXTRAS=&#39;-L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; EXE=&#39;test_listener_event_process_state&#39; CFLAGS_EXTRAS=&#39;-std=c++11  -I/usr/local/src/lldb/include&#39; CXX_SOURCES=&#39;driver.cpp listener_test.cpp test_listener_event_process_state.cpp&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>driver.cpp:22: error: undefined reference to &#39;lldb::SBDebugger::Initialize()&#39;<br></p><p>driver.cpp:23: error: undefined reference to &#39;lldb::SBDebugger::Create()&#39;<br></p><p>driver.cpp:26: error: undefined reference to &#39;lldb::SBDebugger::IsValid() const&#39;<br></p><p>driver.cpp:36: error: undefined reference to &#39;lldb::SBDebugger::Destroy(lldb::SBDebugger&amp;)&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>driver.cpp:38: error: undefined reference to &#39;lldb::SBDebugger::~SBDebugger()&#39;<br></p><p>listener_test.cpp:33: error: undefined reference to &#39;lldb::SBDebugger::CreateTarget(char const*)&#39;<br></p><p>listener_test.cpp:34: error: undefined reference to &#39;lldb::SBTarget::IsValid() const&#39;<br></p><p>listener_test.cpp:36: error: undefined reference to &#39;lldb::SBTarget::BreakpointCreateByName(char const*, char const*)&#39;<br></p><p>listener_test.cpp:37: error: undefined reference to &#39;lldb::SBBreakpoint::IsValid() const&#39;<br></p><p>listener_test.cpp:41: error: undefined reference to &#39;lldb::SBError::SBError()&#39;<br></p><p>listener_test.cpp:42: error: undefined reference to &#39;lldb::SBTarget::Launch(lldb::SBListener&amp;, char const**, char const**, char const*, char const*, char const*, char const*, unsigned int, bool, lldb::SBError&amp;)&#39;<br></p><p>listener_test.cpp:48: error: undefined reference to &#39;lldb::SBError::Success() const&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBError::~SBError()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBBreakpoint::~SBBreakpoint()&#39;<br></p><p>listener_test.cpp:69: error: undefined reference to &#39;lldb::SBTarget::~SBTarget()&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::SBListener(char const*)&#39;<br></p><p>listener_test.cpp:21: error: undefined reference to &#39;lldb::SBListener::~SBListener()&#39;<br></p><p>test_listener_event_process_state.cpp:28: error: undefined reference to &#39;lldb::SBEvent::SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:29: error: undefined reference to &#39;lldb::SBListener::WaitForEvent(unsigned int, lldb::SBEvent&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:31: error: undefined reference to &#39;lldb::SBEvent::IsValid() const&#39;<br></p><p>test_listener_event_process_state.cpp:35: error: undefined reference to &#39;lldb::SBProcess::GetProcessFromEvent(lldb::SBEvent const&amp;)&#39;<br></p><p>test_listener_event_process_state.cpp:36: error: undefined reference to &#39;lldb::SBStream::SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:38: error: undefined reference to &#39;lldb::SBProcess::GetNumThreads()&#39;<br></p><p>test_listener_event_process_state.cpp:40: error: undefined reference to &#39;lldb::SBStream::Clear()&#39;<br></p><p>test_listener_event_process_state.cpp:41: error: undefined reference to &#39;lldb::SBProcess::GetThreadAtIndex(unsigned long)&#39;<br></p><p>test_listener_event_process_state.cpp:42: error: undefined reference to &#39;lldb::SBThread::GetDescription(lldb::SBStream&amp;) const&#39;<br></p><p>test_listener_event_process_state.cpp:43: error: undefined reference to &#39;lldb::SBStream::GetData()&#39;<br></p><p>test_listener_event_process_state.cpp:46: error: undefined reference to &#39;lldb::SBThread::GetNumFrames()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBThread::GetFrameAtIndex(unsigned int)&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::GetSymbol() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::GetName() const&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBSymbol::~SBSymbol()&#39;<br></p><p>test_listener_event_process_state.cpp:48: error: undefined reference to &#39;lldb::SBFrame::~SBFrame()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:52: error: undefined reference to &#39;lldb::SBThread::~SBThread()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBStream::~SBStream()&#39;<br></p><p>test_listener_event_process_state.cpp:53: error: undefined reference to &#39;lldb::SBProcess::~SBProcess()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>test_listener_event_process_state.cpp:54: error: undefined reference to &#39;lldb::SBEvent::~SBEvent()&#39;<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [test_listener_event_process_state] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/api/multithreaded<br></p><p>----------------------------------------------------------------------<br></p><p>Ran 12 tests in 41.191s<br></p><p><br></p><p><br>RESULT: FAILED (0 passes, 0 failures, 9 errors, 0 skipped, 3 expected failures, 0 unexpected successes)<br></p><p>Session logs for test failures/errors/unexpected successes can be found in directory &#39;2016-08-28-05_59_45&#39;<br></p><p><br></p><p><br>[TestMultithreaded.py FAILED]<br></p><p>Command invoked: /usr/bin/python /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 45531 --inferior -p TestMultithreaded.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p>1 out of 4 test suites processed - TestMultithreaded.py<br></p><p><br></p><p><br>Session logs for test failures/errors/unexpected successes will go into directory &#39;2016-08-28-05_59_45&#39;<br></p><p>Command invoked: /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 45531 --inferior -p TestPluginCommands.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p><br></p><p><br>Configuration: arch=x86_64 compiler=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang<br></p><p>----------------------------------------------------------------------<br></p><p>Collected 1 test<br></p><p><br></p><p><br>======================================================================<br></p><p>ERROR: test_load_plugin (TestPluginCommands.PluginCommandTestCase)<br></p><p>   Test that plugins that load commands work correctly.<br></p><p>----------------------------------------------------------------------<br></p><p>Error when building test subject.<br></p><p><br></p><p><br>Build Command:<br></p><p>make ARCH=x86_64  CC=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang SWIFTCC=/usr/local/src/build/buildbot_linux/swift-linux-x86_64/bin/swiftc DYLIB_CXX_SOURCES=&#39;plugin.cpp&#39; LD_EXTRAS=&#39;-shared -L/usr/local/src/build/buildbot_linux/lldb-linux-x86_64/lib64/python2.7/site-packages/../.. -llldb&#39; DYLIB_NAME=&#39;plugin&#39; CFLAGS_EXTRAS=&#39;-std=c++11 -I/usr/local/src/lldb/include -fPIC&#39;<br></p><p><br></p><p><br>Build Command Output:<br></p><p>/usr/bin/ld: error: cannot find -llldb<br></p><p>clang-3.9: error: linker command failed with exit code 1 (use -v to see invocation)<br></p><p>make: *** [libplugin.so] 错误 1<br></p><p><br></p><p><br>Test Directory:<br></p><p>/usr/local/src/lldb/packages/Python/lldbsuite/test/functionalities/plugins/commands<br></p><p>----------------------------------------------------------------------<br></p><p>Ran 1 test in 0.430s<br></p><p><br></p><p><br>RESULT: FAILED (0 passes, 0 failures, 1 errors, 0 skipped, 0 expected failures, 0 unexpected successes)<br></p><p>Session logs for test failures/errors/unexpected successes can be found in directory &#39;2016-08-28-05_59_45&#39;<br></p><p><br></p><p><br>[TestPluginCommands.py FAILED]<br></p><p>Command invoked: /usr/bin/python /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 45531 --inferior -p TestPluginCommands.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p>2 out of 4 test suites processed - TestPluginCommands.py<br></p><p><br></p><p><br>Session logs for test failures/errors/unexpected successes will go into directory &#39;2016-08-28-05_59_45&#39;<br></p><p>Command invoked: /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 45531 --inferior -p TestREPLThrowReturn.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p><br></p><p><br>Configuration: arch=x86_64 compiler=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang<br></p><p>----------------------------------------------------------------------<br></p><p>Collected 1 test<br></p><p><br></p><p><br></p><p><br></p><p>[TestREPLThrowReturn.py FAILED]<br></p><p>Command invoked: /usr/bin/python /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 45531 --inferior -p TestREPLThrowReturn.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p>3 out of 4 test suites processed - TestREPLThrowReturn.py<br></p><p><br></p><p><br>Session logs for test failures/errors/unexpected successes will go into directory &#39;2016-08-28-05_59_45&#39;<br></p><p>Command invoked: /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 45531 --inferior -p TestFormatters.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p><br></p><p><br>Configuration: arch=x86_64 compiler=/usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang<br></p><p>----------------------------------------------------------------------<br></p><p>Collected 3 tests<br></p><p><br></p><p><br>python: /usr/local/src/llvm/tools/clang/lib/AST/Decl.cpp:2562: bool clang::FunctionDecl::isReservedGlobalPlacementOperator() const: Assertion `getDeclName().getNameKind() == DeclarationName::CXXOperatorName&#39; failed.<br></p><p><br></p><p><br>[TestFormatters.py FAILED]<br></p><p>Command invoked: /usr/bin/python /usr/local/src/lldb/test/dotest.py --executable /usr/local/src/build/buildbot_linux/lldb-linux-x86_64/bin/lldb --rerun-all-issues -C /usr/local/src/build/buildbot_linux/llvm-linux-x86_64/bin/clang -s 2016-08-28-05_59_45 --results-port 45531 --inferior -p TestFormatters.py /usr/local/src/lldb/packages/Python/lldbsuite/test --event-add-entries worker_index=0:int<br></p><p>4 out of 4 test suites processed - TestFormatters.py     <br></p><p>Test rerun complete<br></p><p><br></p><p><br></p><p><br></p><p>=============<br></p><p>Issue Details<br></p><p>=============<br></p><p>ERROR: test_breakpoint_callback_dwarf (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: test_breakpoint_callback_dwo (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: test_breakpoint_callback_gmodules (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: test_load_plugin (functionalities/plugins/commands/TestPluginCommands.py)<br></p><p>ERROR: test_sb_api_listener_event_description_dwarf (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: test_sb_api_listener_event_description_dwo (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: test_sb_api_listener_event_description_gmodules (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: test_sb_api_listener_event_process_state_dwarf (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: test_sb_api_listener_event_process_state_dwo (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: test_sb_api_listener_event_process_state_gmodules (api/multithreaded/TestMultithreaded.py)<br></p><p>ERROR: [EXCEPTIONAL EXIT 6 (SIGABRT)] test_dwarf (expression_command/formatters/TestFormatters.py)<br></p><p>ERROR: [EXCEPTIONAL EXIT 11 (SIGSEGV)] test_with_dwarf (repl/error_return/TestREPLThrowReturn.py)<br></p><p>UNEXPECTED SUCCESS: testREPL (repl/structs/TestREPLStructs.py)<br></p><p>UNEXPECTED SUCCESS: testREPL (repl/pounwrapping/TestPOUnwrapping.py)<br></p><p>UNEXPECTED SUCCESS: testREPL (repl/dictionary/TestREPLDictionary.py)<br></p><p>UNEXPECTED SUCCESS: testREPL (repl/int_vars/TestREPLIntVars.py)<br></p><p>UNEXPECTED SUCCESS: test_Hc_then_Csignal_signals_correct_thread_launch_llgs (tools/lldb-server/TestLldbGdbServer.py)<br></p><p>UNEXPECTED SUCCESS: test_and_python_api_dwarf (lang/c/bitfields/TestBitfields.py)<br></p><p>UNEXPECTED SUCCESS: test_and_python_api_dwo (lang/c/bitfields/TestBitfields.py)<br></p><p>UNEXPECTED SUCCESS: test_and_python_api_gmodules (lang/c/bitfields/TestBitfields.py)<br></p><p>UNEXPECTED SUCCESS: test_and_run_command_dwarf (lang/c/const_variables/TestConstVariables.py)<br></p><p>UNEXPECTED SUCCESS: test_and_run_command_dwo (lang/c/const_variables/TestConstVariables.py)<br></p><p>UNEXPECTED SUCCESS: test_and_run_command_gmodules (lang/c/const_variables/TestConstVariables.py)<br></p><p>UNEXPECTED SUCCESS: test_any_type_dwarf (lang/swift/any/TestSwiftAnyType.py)<br></p><p>UNEXPECTED SUCCESS: test_any_type_dwo (lang/swift/any/TestSwiftAnyType.py)<br></p><p>UNEXPECTED SUCCESS: test_any_type_gmodules (lang/swift/any/TestSwiftAnyType.py)<br></p><p>UNEXPECTED SUCCESS: test_command_script_immediate_output_dwarf (functionalities/command_script_immediate_output/TestCommandScriptImmediateOutput.py)<br></p><p>UNEXPECTED SUCCESS: test_command_script_immediate_output_dwo (functionalities/command_script_immediate_output/TestCommandScriptImmediateOutput.py)<br></p><p>UNEXPECTED SUCCESS: test_command_script_immediate_output_gmodules (functionalities/command_script_immediate_output/TestCommandScriptImmediateOutput.py)<br></p><p>UNEXPECTED SUCCESS: test_dwarf (functionalities/tsan/thread_leak/TestTsanThreadLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_dwarf (functionalities/asan/TestReportData.py)<br></p><p>UNEXPECTED SUCCESS: test_dwarf (functionalities/asan/TestMemoryHistory.py)<br></p><p>UNEXPECTED SUCCESS: test_dwarf (lang/swift/file_private/TestFilePrivate.py)<br></p><p>UNEXPECTED SUCCESS: test_dwo (functionalities/tsan/thread_leak/TestTsanThreadLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_dwo (functionalities/asan/TestMemoryHistory.py)<br></p><p>UNEXPECTED SUCCESS: test_dwo (functionalities/asan/TestReportData.py)<br></p><p>UNEXPECTED SUCCESS: test_dwo (lang/swift/file_private/TestFilePrivate.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_basic_dwarf (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_basic_dwo (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_basic_gmodules (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_log_dwarf (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_log_dwo (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_log_gmodules (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_multitarget_dwarf (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_multitarget_dwo (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_fd_leak_multitarget_gmodules (functionalities/avoids-fd-leak/TestFdLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_get_value_as_unsigned_sbapi_dwarf (lang/swift/get_value/TestSwiftGetValueAsUnsigned.py)<br></p><p>UNEXPECTED SUCCESS: test_get_value_as_unsigned_sbapi_dwo (lang/swift/get_value/TestSwiftGetValueAsUnsigned.py)<br></p><p>UNEXPECTED SUCCESS: test_get_value_as_unsigned_sbapi_gmodules (lang/swift/get_value/TestSwiftGetValueAsUnsigned.py)<br></p><p>UNEXPECTED SUCCESS: test_gmodules (functionalities/tsan/thread_leak/TestTsanThreadLeak.py)<br></p><p>UNEXPECTED SUCCESS: test_gmodules (functionalities/asan/TestReportData.py)<br></p><p>UNEXPECTED SUCCESS: test_gmodules (functionalities/asan/TestMemoryHistory.py)<br></p><p>UNEXPECTED SUCCESS: test_gmodules (lang/swift/file_private/TestFilePrivate.py)<br></p><p>UNEXPECTED SUCCESS: test_lldbmi_break_insert_function (tools/lldb-mi/breakpoint/TestMiBreak.py)<br></p><p>UNEXPECTED SUCCESS: test_lldbmi_break_insert_function_pending (tools/lldb-mi/breakpoint/TestMiBreak.py)<br></p><p>UNEXPECTED SUCCESS: test_lldbmi_gdb_set_target_async_off (tools/lldb-mi/TestMiGdbSetShow.py)<br></p><p>UNEXPECTED SUCCESS: test_lldbmi_process_output (tools/lldb-mi/syntax/TestMiSyntax.py)<br></p><p>UNEXPECTED SUCCESS: test_lldbmi_settings_set_target_run_args_after (tools/lldb-mi/interpreter/TestMiInterpreterExec.py)<br></p><p>UNEXPECTED SUCCESS: test_process_interrupt_dwarf (functionalities/thread/state/TestThreadStates.py)<br></p><p>UNEXPECTED SUCCESS: test_process_interrupt_dwo (functionalities/thread/state/TestThreadStates.py)<br></p><p>UNEXPECTED SUCCESS: test_process_interrupt_gmodules (functionalities/thread/state/TestThreadStates.py)<br></p><p>UNEXPECTED SUCCESS: test_restart_bug_dwarf (functionalities/signal/raise/TestRaise.py)<br></p><p>UNEXPECTED SUCCESS: test_restart_bug_dwo (functionalities/signal/raise/TestRaise.py)<br></p><p>UNEXPECTED SUCCESS: test_restart_bug_gmodules (functionalities/signal/raise/TestRaise.py)<br></p><p>UNEXPECTED SUCCESS: test_scope_lookup_before_using_with_run_command_dwarf (lang/cpp/namespace/TestNamespaceLookup.py)<br></p><p>UNEXPECTED SUCCESS: test_scope_lookup_before_using_with_run_command_dwo (lang/cpp/namespace/TestNamespaceLookup.py)<br></p><p>UNEXPECTED SUCCESS: test_scope_lookup_before_using_with_run_command_gmodules (lang/cpp/namespace/TestNamespaceLookup.py)<br></p><p>UNEXPECTED SUCCESS: test_specialized_typedef_from_pch_gmodules (lang/cpp/gmodules/TestWithModuleDebugging.py)<br></p><p>UNEXPECTED SUCCESS: test_step_back_out_dwarf (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_step_back_out_dwo (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_step_back_out_gmodules (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_step_out_dwarf (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_step_out_dwo (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_step_out_gmodules (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_step_over_dwarf (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_step_over_dwo (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_step_over_gmodules (functionalities/longjmp/TestLongjmp.py)<br></p><p>UNEXPECTED SUCCESS: test_watched_var_should_only_hit_when_in_scope_dwarf (functionalities/watchpoint/variable_out_of_scope/TestWatchedVarHitWhenInScope.py)<br></p><p>UNEXPECTED SUCCESS: test_watched_var_should_only_hit_when_in_scope_dwo (functionalities/watchpoint/variable_out_of_scope/TestWatchedVarHitWhenInScope.py)<br></p><p>UNEXPECTED SUCCESS: test_watched_var_should_only_hit_when_in_scope_gmodules (functionalities/watchpoint/variable_out_of_scope/TestWatchedVarHitWhenInScope.py)<br></p><p><br></p><p><br>===================<br></p><p>Test Result Summary<br></p><p>===================<br></p><p>Test Methods:       2788<br></p><p>Reruns:               15<br></p><p>Success:            1749<br></p><p>Expected Failure:    150<br></p><p>Failure:               0<br></p><p>Error:                10<br></p><p>Exceptional Exit:      2<br></p><p>Unexpected Success:   68<br></p><p>Skip:                809<br></p><p>Timeout:               0<br></p><p>Expected Timeout:      0<br></p><p>./swift/utils/build-script: fatal error: command terminated with a non-zero exit status 1, aborting<br></p><p> <br></p><p>./swift/utils/build-script: fatal error: command terminated with a non-zero exit status 1, aborting<br></p><p><br></p><p>===================================================<br></p><p><br>How can I solve?<br>-------------- next part --------------<br>An HTML attachment was scrubbed...<br>URL: &lt;https://lists.swift.org/pipermail/swift-dev/attachments/20160829/bedb5548/attachment.html&gt;<br></p></div></li></ul></div>    </main>
    <script src="/js/app-c283ee129de63ad743722e9511e67a5d.js?vsn=d"></script>
  </body>
  <footer>
    <p>Swift and the Swift logo are trademarks of Apple Inc.</p>
  </footer>
</html>
